### PR TITLE
Feature/3135/cut off messages

### PR DESCRIPTION
**Pull request by agduncan94**
https://github.com/dockstore/dockstore/issues/3135
Edit: Ticket has been clarified to also improve the user experience when using the manual registration wizard. I've added helper text as shown in the first image. I've also added more information to the following pages after user has selected the registration type, which can be seen in the second image.

I am open to comments on the text itself too.

workflow-reg-pg-1
workflow-reg-pg2

### CODE REVIEW

**Comment by garyluu** [comment]
Codacy, ng-deep deprecated

File: src/app/mytools/my-tool/my-tool.component.html [hunk 1]
```diff
@@ -82,7 +82,7 @@
                   will be synchronized in the linking process.
                 </li>
                 <li>
-                  Press the 'Add Tool' button to manually add a tool.
+                  Press the 'Add Tool' button to manually add a tool usnig the registration wizard.
```
Comment (denis-yuen) [inline src/app/mytools/my-tool/my-tool.component.html]: Typo

File: src/app/workflow/register-workflow-modal/register-workflow-modal.component.html [hunk 2]
```diff
@@ -62,6 +67,10 @@ <h4 mat-dialog-title>Register Workflow</h4>
         (submit)="registerWorkflow()"
         novalidate
       >
+        <p>
+          Manually register a workflow with custom values. Use the optional workflow name field to handle repositories with multiple
```
Comment (denis-yuen) [inline src/app/workflow/register-workflow-modal/register-workflow-modal.component.html]: Hmmm, perhaps `Manually register a workflow with custom file paths`. `Values` seems a bit meaningless on its own.

File: src/app/workflow/register-workflow-modal/register-workflow-modal.component.ts [hunk 3]
```diff
@@ -62,15 +62,18 @@ export class RegisterWorkflowModalComponent implements OnInit, AfterViewChecked,
...
     {
-      label: 'Quickly register workflows from GitHub, Bitbucket, and GitLab',
+      label: 'Quickly register remote workflows',
+      extendedLabel: 'Toggle repositories from GitHub, Bitbucket, and GitLab to quickly create workflows on Dockstore.',
       value: 0
     },
     {
-      label: 'Create non-standard workflows from GitHub, Bitbucket, and GitLab',
+      label: 'Register custom remote workflows',
+      extendedLabel: 'Manually add individual workflows with custom metadata from repositories on GitHub, Bitbucket, and GitLab.',
```
Comment (denis-yuen) [inline src/app/workflow/register-workflow-modal/register-workflow-modal.component.ts]: Same idea, also "values" are used above but "metadata" is used here. 
Maybe  `Manually add individual workflows at custom file paths` or `Manually add individual workflows at customized locations in repositories on GitHub, Bitbucket, and GitLab`

If using `locations` here, use it above. Same deal if using `file paths`.

File: src/styles.scss [hunk 4]
```diff
@@ -1318,3 +1318,6 @@ mat-icon.star {
 #tool-path {
   overflow-wrap: break-word;
 }
+.text-wrap {
+  white-space: normal !important;
```
Comment (garyluu) [inline src/styles.scss]: The `!important` appears redundant.

File: src/app/container/register-tool/register-tool.component.html [hunk 5]
```diff
@@ -359,6 +367,10 @@ <h4 mat-dialog-title>Register Tool</h4>
         novalidate
       >
         <div class="modal-body">
+          <p>
+            Create a tool with all descriptor files stored and edited on Dockstore.org. Docker images are stored on sites like Quay.io and
+            DockerHub.
```
Comment (coverbeck) [inline src/app/container/register-tool/register-tool.component.html]: The images can be stored in any Docker image repo, can't they?

Comment (agduncan94) [reply src/app/container/register-tool/register-tool.component.html]: Yes they can, I was just trying to list two as examples. Though if that isn't obvious I can try and change the wording.

> Create a tool with all descriptor files stored and edited on Dockstore.org. Docker images are stored on sites **_such as_** Quay.io and DockerHub

Comment (agduncan94) [reply src/app/container/register-tool/register-tool.component.html]: Well not any Docker image repo, there are a select few we support (though easyish to add more).

Comment (coverbeck) [reply src/app/container/register-tool/register-tool.component.html]: Whoops, the way it was wrapping the right side was clipped, and I only saw 

```
...Docker images are stored on 
Dockerhub
```
And totally didn't see `are stored on sites like Quay.io and`. So it was and is good. Sorry for the confusion.

**Pull request merged**